dadguide/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .dadguide import Dadguide
+from .dadguide import *
```

Comment:
?

Reply:
Necessary for PDE and also for testing things out with ^eval